client/state/utils/README.md

```diff
@@ -0,0 +1,18 @@
+Utils
+========
+
+A module for those little helpers that need to interact woth the store.
+
+# Actions
+Used in combination with the Redux store instance `dispatch` function, actions can be used in manipulating the current global state.
+
+## uniqueId( prefix )
+Meant as a replacement for _ and lodash "uniqueId" function.
+Returns unique numeric prefixed by prefix and increments internal state, so the next uniqueId will be different.
```

Comment:
@mcsf 
> here I'm getting at is that we might want a sort of standard way to bind some stateful function (uniqueId, memoize) to a React tree/context/request

Thats the whole point of this `utils` state subtree to hold this kind of functions. Maybe memoize is not the best example, because maybe we dont want to store cache in the store..